api/graphql/mutations/project.py

```diff
@@ -34,3 +52,108 @@ async def add_comment(
             entity=CommentEntityType.project, entity_id=id, content=content
         )
         return GraphQLComment.from_internal(result)
+
+    @strawberry.mutation
+    async def create_project(
+        self,
+        name: str,
+        dataset: str,
+        create_test_project: bool,
+        info: Info,
+    ) -> int:
+        """
+        Create a new project
+        """
+        connection = info.context['connection']
+        ptable = ProjectPermissionsTable(connection)
+
+        await ptable.check_project_creator_permissions(author=connection.author)
+
+        pid = await ptable.create_project(
+            project_name=name,
+            dataset_name=dataset,
+            author=connection.author,
+        )
+
+        if create_test_project:
+            await ptable.create_project(
+                project_name=name + '-test',
+                dataset_name=dataset,
+                author=connection.author,
+            )
+
+        return pid
+
+    @strawberry.mutation
+    async def update_project(
+        self,
+        project: str,
+        project_update_model: JSON,
+        info: Info,
+    ) -> JSON:
+        """Update a project by project name"""
+        connection = info.context['connection']
+        connection.check_access({ProjectMemberRole.project_admin})
```

Comment:
Same issue here, can't use `check_access` as it requires a project on the connection.

Reply:
Fixed!